param: {
    # Controller parameters
    k_ctrl_xy: [-9.0, -6.0],
    k_ctrl_z: [-25.0, -10.0],
    k_ctrl_att: [-9.0, -6.0],
    k_ctrl_yaw: [36.0, 12.0],

    # Parameter Estimator
    a_x0: 0,
    a_y0: 0,
    b_x0: 4.0,
    b_y0: -4.0,

    a2_0: [-16.0, 0, 0, 0],
    b2_0:[5.0, 5.0, 5.0, 5.0,
            -50.0, -50.0, 50.0, 50.0,
            -50.0, 50.0, 50.0, -50.0,
            50.0, -50.0, 50.0, -50.0],

    # Gains
    k_par_x: [0.001, 0.01],
    k_par_y: [0.001, 0.01],
    k_par_a2: [0.002, 0.002, 0.002, 0.002],
    k_par_b2: [0.0001, 0.0001, 0.0001, 0.0001,
                0.0001, 0.0001, 0.0001, 0.0001,
                0.0001, 0.0001, 0.0001, 0.0001,
                0.0001, 0.0001, 0.0001, 0.0001], 
    # Limits
    betax_lim: [1.0, 18.0],
    betay_lim: [-18.0, -1.0],
    b2d_lbound: [1.0, 1.0, 1.0, 1.0,
                -100.0, -100.0, 1.0, 1.0,
                -100.0, 1.0, 1.0, -100.0,
                1.0, -100.0, 1.0, -100.0],
    b2d_ubound: [10.0, 10.0, 10.0, 10.0,
                -1.0, -1.0, 100.0, 100.0,
                -1.0, 100.0, 100.0, -1.0,
                100.0, -1.0, 100.0, -1.0]
}
